google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -479,6 +494,18 @@ def deploy_to_local_endpoint(
         envs = {env.name: env.value for env in self.serving_container_spec.env}
         ports = [port.container_port for port in self.serving_container_spec.ports]
 
+        if gpu_count and gpu_device_ids:
+            raise ValueError(
+                "Either gpu_count or gpu_device_ids can be set but both are set."
+            )
+        if ((gpu_count or gpu_device_ids) and gpu_capabilities is None) or (
```

Comment:
Are we just checking for None here? or should an empty list also raise the error?

Reply:
It can be empty list.